hrm-service/src/permissions/permissions-routes-v0.ts

```diff
@@ -0,0 +1,24 @@
+import { SafeRouter, publicEndpoint } from '../permissions';
+import { rulesMap, getPermissionsConfigName } from '../permissions';
+// eslint-disable-next-line prettier/prettier
+import type { Request, Response } from 'express';
+import createError from 'http-errors';
+
+const permissionsRouter = SafeRouter();
+
+permissionsRouter.get('/', publicEndpoint, (req: Request, res: Response, next) => {
```

Comment:
Do we want this to be entirely public? While right now it's already public because the source is public, we may not keep all helpline configuration public forever. Do you still need a valid Flex JWE for an account in order for this to authenticate?

Reply:
`publicEndpoint` is actually a poor name, because this is the middleware used to "authorize" any request (no specific permissions needed). But yes, it gets to this point only after the "authentication" (valid JWT should pass through).
In fact in this endpoint, we do not receive any parameters because we are stripping the `accountSid` from the request itself and then using it to compose the proper permissions config set for this account in envars (they are in the form of `PERMISSIONS_<accountSid>=<permissionsConfigName>` where `permissionsConfigName` is like `zm`, `br`, etc). If the account does not have permissions set or they are invalid we send a 5xx (that's what I was asking Steve above, if a 4xx makes more sense).